docs/fundamentals/wadm/deploying.md

```diff
@@ -2,13 +2,211 @@
+instances of wadm.
+
+### Single lattice management
+
+The simplest of all deployment patterns is to run a group of wadm instances per lattice. Credentials
+for this only need to be able to receive messages from a single lattice and will only ever manage
+one lattice. If you are just starting out with wasmCloud, this is most likely the deployment pattern
+you will want to use. See [Configuring NATS Credentials](#configuring-nats-credentials) for more
+information on how to configure NATS credentials.
+
+### Multi-lattice management
+
+This pattern is slightly more complex than the single lattice management pattern. In this pattern,
+you are managing multiple lattices all running on the same NATS cluster (e.g. such as for your team
+or group), meaning wadm will be looking for all events on `wasmbus.evt.*`. This is useful if you
+have multiple lattices running in different environments (e.g. dev, staging, production) and want to
+manage them all from a single wadm instance. This pattern requires that you configure NATS
+credentials to be able to receive messages from all lattices. See [Configuring NATS
+Credentials](#configuring-nats-credentials) for more information on how to configure NATS
+credentials. 
+
+However, it is important to note that the difference between this pattern and the single lattice
+management pattern is purely in the credentials used (you still can run the instances in the same
+way). The wadm instances themselves are identical. This means that you can start with a single
+lattice management pattern and then move to a multi-lattice management pattern by simply changing
+the credentials used by wadm.
+
+### Multi-tenant management
+
+The most complex case is when you want to manage multiple lattices running on different NATS
+clusters. This is useful if you are running wasmCloud for various groups/teams across an
+organization and want to manage multiple lattices running on clusters with segmented traffic. This
+pattern requires that you configure NATS credentials to be able to receive messages from all
+lattices. 
+
+:::warning
+Currently, this pattern is not supported. We are working on adding support for this in the 0.5
+release with detailed instructions on how it could be configured.
+:::
+
+## Configuring NATS Credentials
+
+Wadm uses NATS for communication with the lattice. This means that you will need to configure NATS
+credentials for wadm to use and for users to authenticate with. Like wasmCloud hosts, wadm expects
+to use decentralized JWT authentication or anonymous authentication. This is a very complex topic
+that will not be covered in full here, but you can get a full walkthrough in the [NATS
+documentation](https://docs.nats.io/running-a-nats-service/nats_admin/security/jwt). In this section
+we will cover what topics wadm needs access to in order to function.
+
+TODO: I need some help here enumerating how to restrict credentials down for people
```

Comment:
@autodidaddict This is where I could use some help filling out the best things to do here. No need to worry about the multi tenant option for now